src/User.java

```diff
@@ -0,0 +1,111 @@
+import java.util.Scanner;
+
+/**
+ * Created by melmo on 11/30/16.
+ */
+public class User {
+    private String name;
+    private int balance;
+    private Scanner input = new Scanner(System.in);
+
+    public User(int startBalance) {
+        this.balance = startBalance;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public int getBalance() {
+        return this.balance;
+    }
+
+    public void chooseName() throws Exception {
+        System.out.println("What is your name?");
+        String name = this.input.nextLine();
+        if (!name.equals("")) {
+            this.name = name;
+            System.out.format("Hello %s.\n", this.name);
+        } else {
+            throw new Exception("Invalid input!");
+        }
+    }
+
+    public void whichOp() throws Exception {
+        System.out.format("What would you like to do?\n" +
+                "1 : Check my balance\n" +
+                "2 : Withdraw funds\n" +
+                "3 : Deposit funds\n" +
+                "4 : Exit\n");
+        String choice = this.input.nextLine();
+        if (choice.equals("1")) {
```

Comment:
When you start having a larger number of if/if else/else statements you should consider a switch statement.  The syntax is a bit less verbose. We don't cover it much in class but you might want to take a gander at the documentation: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html